tools/accuracy_checker/accuracy_checker/metrics/classification.py

```diff
@@ -671,3 +671,56 @@ def reset(self):
     @classmethod
     def get_common_meta(cls):
         return {'target': 'higher-worse'}
+
+
+class URLNetAccuracy(Metric):
+    __provider__ = 'urlnet_score'
```

Comment:
could you please provide more clear metric name, this approach is not related to specific model

Reply:
renamed to class to UrlCassificationScore and __provider__ to url_classification_score